mlir/test/fusion/xdlops/tosa-to-miopen-relu.x2.e2e.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: mlir-miopen-driver -host-pipeline highlevel %s | miopen-gen -ph -print-results -rand 1 -rand_type float - | mlir-miopen-driver -c  | mlir-rocm-runner --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s
+
+module {
+// CHECK: Unranked Memref base
```

Comment:
can we be more specific about the expected test result?

also can a lowering test be added to check the expected IR without running the kernel?

Reply:
I suppose this is e2e test verifying the result against that from CPU, so expecting correct result of fused convolution and relu.

Lowering test is good point, I'll add it with specific ir and corresponding result.